### PR TITLE
[ZH] Timelines

### PR DESCRIPTION
**Pull request by Sisters10086**
Halfway finished, copied some elements from https://osu.ppy.sh/help/wiki/Beatmap_Editor/Compose#top-left-hit-objects-timeline for consistency. First time translating, feel free to comment/give suggestions.

### CODE REVIEW

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# 时间轴
+
+在[铺面编辑器](/wiki/beatmap_editor)中，铺面制作者们将会遇到三种不同的时间线。
```
Comment (Robwolver) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 铺面>谱面，下同

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+_在[快捷键](/wiki/shortcut_key_reference/#general)中可以找到有关时间轴的一系列快捷键。_
+
+## 音乐播放器
+
+![Song's Timeline](/wiki/shared/BE_STL.jpg "Song's Timeline")
+
+在铺面编辑器中的任何一部分都能看见音乐播放器。
+
+以毫秒为单位的时间戳与歌曲进程百分比都在播放器的左侧
+如果一个铺面拥有storyboard，这百分比可能会显示“前奏”或“结尾”。
```
Comment (Robwolver) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 注意翻译规则

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+| 灰色 (突出部分) | 休息时间 |
+| 橙色 (突出部分) | Kiai time（timing切换点？） |
+
+
+
+## 游戏物件
+
+根据游戏模式的不同会有共两种不同的物件时间轴。
+
+### osu!, osu!太鼓, 和osu!接水果
```
Comment (Robwolver) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 专有名词暂时不翻译

File: wiki/Beatmap_Editor/Timelines/zh.md (hunk 2 above)
Comment (Robwolver) [inline wiki/Beatmap_Editor/Timelines/zh.md]: storyboard 前应该有空格，首字母大写。下同

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+![在osu!、osu!太鼓、osu!接水果中的物件时间轴](/wiki/shared/BE_NTL.jpg "这显示了相对于音符时值与时间戳的物件。")
```
Comment (Robwolver) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 同上

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+![Song's Timeline](/wiki/shared/BE_STL.jpg "Song's Timeline")
+
+在铺面编辑器中的任何一部分都能看见音乐播放器。
+
+以毫秒为单位的时间戳与歌曲进程百分比都在播放器的左侧
+如果一个铺面拥有storyboard，这百分比可能会显示“前奏”或“结尾”。
+
+被标记的时间轴和强制播放音乐按钮在播放器的中间。
+“Test"按键会保存目前的作图进度，同时会从目前的时间戳来开始试玩。
```
Comment (kj415j45) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 用`` `Test` ``
参考 #347 ASG

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 6]
```diff
@@ -30,17 +30,17 @@ _在[快捷键](/wiki/shortcut_key_reference/#general)中可以找到有关时
...
+| 橙色 (突出部分) | Kiai time |
 
 
 
 ## 游戏物件
 
 根据游戏模式的不同会有共两种不同的物件时间轴。
 
-### osu!, osu!太鼓, 和osu!接水果
+### osu!, osu!Taiko, 和osu!CatchTheBeat
```
Comment (kj415j45) [inline wiki/Beatmap_Editor/Timelines/zh.md]: `osu!taiko`  `osu!catch`

Comment (kj415j45) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 另外注意空格

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 7]
```diff
@@ -30,17 +30,17 @@ _在[快捷键](/wiki/shortcut_key_reference/#general)中可以找到有关时
...
 
-![在osu!、osu!太鼓、osu!接水果中的物件时间轴](/wiki/shared/BE_NTL.jpg "这显示了相对于音符时值与时间戳的物件。")
+![在osu!、osu!Taiko、osu!CatchTheBeat中的物件时间轴](/wiki/shared/BE_NTL.jpg "这显示了相对于音符时值与时间戳的物件。")
```
Comment (kj415j45) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 同上，注意空格

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 8]
```diff
@@ -11,13 +11,13 @@ _在[快捷键](/wiki/shortcut_key_reference/#general)中可以找到有关时
...
-在铺面编辑器中的任何一部分都能看见音乐播放器。
+在谱面编辑器中的任何一部分都能看见音乐播放器。
 
 以毫秒为单位的时间戳与歌曲进程百分比都在播放器的左侧
-如果一个铺面拥有storyboard，这百分比可能会显示“前奏”或“结尾”。
+如果一个谱面拥有 Storyboard，这百分比可能会显示“前奏”或“结尾”。
 
 被标记的时间轴和强制播放音乐按钮在播放器的中间。
-“Test"按键会保存目前的作图进度，同时会从目前的时间戳来开始试玩。
+'Test'按键会保存目前的作图进度，同时会从目前的时间戳来开始试玩。
```
Comment (kj415j45) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 是`` `Test` ``不是 `'Test'`

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # 时间轴
 
-在[铺面编辑器](/wiki/beatmap_editor)中，铺面制作者们将会遇到三种不同的时间线。
+在[谱面编辑器](/wiki/beatmap_editor)中，谱面制作者们将会遇到三种不同的时间线。
 这篇文章将逐一介绍它们的作用
```
Comment (kj415j45) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 结尾句号，把这一行和第三行连起来（还没写进asg但是ppy很不爽这种写法，所以还是按他说的做吧）

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 10]
```diff
@@ -59,7 +59,7 @@ _在[快捷键](/wiki/shortcut_key_reference/#general)中可以找到有关时
 
 在osu!mania的作曲模式中，这条时间轴会显示于游戏界面的下方。
 	
-左方的方框表示铺面的物件密集度。
+左方的方框表示谱面的物件密集度。
 
 这被用来当做时间轴。
 中间的方框表示游戏区域
```
Comment (kj415j45) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 同上

**Comment by kj415j45** [comment]
另外请让commit message有意义，不要全都是Update zh.md 🙃

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+### osu!mania
+	
+![osu!mania 游戏界面](/wiki/shared/BEM_PF.jpg "osu!mania 游戏界面")
+
+在osu!mania的作曲模式中，这条时间轴会显示于游戏界面的下方。
+	
+左方的方框表示谱面的物件密集度。
+
+这被用来当做时间轴。
```
Comment (Gennadiyev) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 这也可以被用来当做时间轴

File: wiki/Beatmap_Editor/Timelines/zh.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
+# 时间轴
+
+在[谱面编辑器](/wiki/beatmap_editor)中，谱面制作者们将会遇到三种不同的时间线。
+这篇文章将逐一介绍它们的作用
```
Comment (benpigchu) [inline wiki/Beatmap_Editor/Timelines/zh.md]: 【句号】

**Comment by sr229** [comment]
You seem to use generic commit messages. Next time your commit messages must explain your changes as stated on the [First-Time Contributor Guide](https://osu.ppy.sh/help/wiki/osu!wiki_contribution_guide#commit-changes)

**Comment by sr229** [comment]
Requesting additional updates here? @Sisters10086

**Comment by kj415j45** [comment]
I suppose nothing will be updated, he is offline(Discord) for 10 days

**Comment by sr229** [comment]
CC @TPGPL or @Repflez  for request for closure since this is a stale PR

**Comment by Ephemeralis** [comment]
Stale, closing. Let me know in #osu-wiki if you want to to begin working on this again.

**Pull request closed without merging**